module_utils/network_lsr/argument_validator.py

```diff
@@ -291,7 +291,6 @@ def _validate_impl(self, value, name):
                     "route table value is {0} but cannot be greater than 4294967295".format(
                         value
                     ),
-                )
```

Comment:
Spurious removal of this line, but it was added back in another commit.

Reply:
Other than this nitpicking, the PR looks good to me.

Reply:
Thanks, fixed it.